files/bootstrap.sh

```diff
@@ -494,6 +494,8 @@ if [[ "$CONTAINER_RUNTIME" = "containerd" ]]; then
     sudo cp -v $CONTAINERD_CONFIG_FILE /etc/eks/containerd/containerd-config.toml
   fi
   echo "$(jq '.cgroupDriver="systemd"' $KUBELET_CONFIG)" > $KUBELET_CONFIG
+  echo "$(jq '.systemReservedCgroup="/system"' $KUBELET_CONFIG)" > $KUBELET_CONFIG
+  echo "$(jq '.kubeReservedCgroup="/runtime"' $KUBELET_CONFIG)" > $KUBELET_CONFIG
   sudo sed -i s,SANDBOX_IMAGE,$PAUSE_CONTAINER,g /etc/eks/containerd/containerd-config.toml
```

Comment:
I think we need to move:
```
sudo mv "${TEMPLATE_DIR}/00-runtime-slice.conf" /etc/systemd/system/containerd.service.d/00-runtime-slice.conf
```
here? Because kubelet is only configured to use this slice when containerd is the `CONTAINER_RUNTIME`.

Reply:
Yes we can move this from the setup to here assuming we can guarantee that Containerd hasn't been started by this point.